Dockerfile

```diff
@@ -1,29 +1,20 @@
 #################################
-# Botiana Dockerfile for Debian #
+# Botiana Dockerfile for Alpine #
 #################################
-
-FROM debian
+FROM python:3.7.4-alpine3.10
 MAINTAINER rpkish
 
-# Configure the basic debian image
-RUN apt-get update && apt-get install -y \
-    aufs-tools \
-    python \
-    python-pip \
-    python-lxml \
-    filters \
-    git \
-    python-yaml \
- && rm -rf /var/lib/apt/lists/* \
- && pip install BeautifulSoup \
- && pip install beautifulsoup4 
+# Build deps
+RUN apk update \
+    && apk add --virtual build-dependencies build-base py3-lxml libxml2-dev libxslt-dev
 
 # Install botiana and copy modified settings and dictionaries 
 RUN mkdir /usr/local/botiana/
-COPY ./botiana.py ./settings.py ./sa.yaml ./keywords.yaml ./requirements.txt /usr/local/botiana/
+COPY ./botiana.py ./settings.py ./requirements.txt ./common.py ./keywords.py ./legacy_modules.py  \
+     ./message_router.py ./settings.py ./slack_commands.py /usr/local/botiana/
 
 # run configuration steps
 RUN pip install -r /usr/local/botiana/requirements.txt
 
 # Fire up botiana
-ENTRYPOINT cd /usr/local/botiana && ./botiana.py
+ENTRYPOINT cd /usr/local/botiana && /usr/local/bin/python3 -u ./botiana.py
```

Comment:
put `WORKDIR /usr/local/botiana` before `ENTRYPOINT`
Also, do you have to use `ENTRYPOINT` over `CMD`?

Reply:
Dockerfile updated, thanks!